### DISCUSSION TITLE
Add fields to reports

### DISCUSSION THREAD

**Discussion by Sponsoredby** [vradarserver/vrs#42] [Ideas]
Hi Andrew, is it possible to add additional fields when generating reports eg flight counts

regards

**Comment by vradarserver** [comment]
@Sponsoredby Sorry for the delay, overlooked this one - yes, you can. It's not very well named - click Menu | Criteria and then you can change or rearrange the fields in the DETAIL and LIST tabs in the same way that you can arrange the field lists on the map page.

Those field lists get used for all reports, not just the one you're currently on.

Note that if the value isn't present for an aircraft then the details panel usually won't show it - e.g. if you add a flights count to the list and detail panels, and then you run a report against an aircraft that you have no flights for, then it'll show a 0 or blank flights count in the aircraft list but probably won't show any flight count at all in the details panel.
